lib/types/device-type-json.ts

```diff
@@ -1,8 +1,9 @@
+import type { Contract } from './contract';
 import type { Dictionary } from '../../typings/utils';
 
 /* types for the /device-types/v1 endppoints */
 
-export interface DeviceType {
+export interface DeviceType extends Contract {
 	slug: string;
```

Comment:
A bunch of the properties here should now be inherited from `Contract`, in which case it'd make sense imo to remove the duplication here

Reply:
Resolved in latest push